www/starter-redirects.json

```diff
@@ -4,7 +4,7 @@
   "/gatsby-tailwind-emotion-starter/": "/muhajirframe/gatsby-tailwind-emotion-starter/",
   "/gatsby-starter-redux-firebase/": "/muhajirframe/gatsby-starter-redux-firebase/",
   "/gatsby-blog-starter-kit/": "/DSchau/gatsby-blog-starter-kit/",
-  "/gatsby-contentful-starter/": "/contentful-userland/gatsby-contentful-starter/",
+  "/gatsby-contentful-starter/": "/contentful/starter-gatsby-blog/",
   "/react-gatsby-firebase-authentication/": "/the-road-to-react-with-firebase/react-gatsby-firebase-authentication/",
```

Comment:
is miss here a `/contentful-userland/gatsby-contentful-starter/` -> `/contentful/starter-gatsby-blog/`?